polygeist/tools/cgeist/Test/Verification/sycl/kernels.cpp

```diff
@@ -12,14 +12,16 @@
 
 #include <sycl/sycl.hpp>
 
+// CHECK-NOT: module
+
 // CHECK: !sycl_accessor_1_i32_read_write_global_buffer = !sycl.accessor<[1, i32, read_write, global_buffer], (!sycl.accessor_impl_device<[1], (!sycl.id<1>, !sycl.range<1>, !sycl.range<1>)>, !llvm.struct<(ptr<i32, 1>)>)>
 // CHECK: !sycl_id_1_ = !sycl.id<1>
 // CHECK: !sycl_item_1_1_ = !sycl.item<[1, true], (!sycl.item_base<[1, true], (!sycl.range<1>, !sycl.id<1>, !sycl.id<1>)>)>
-// CHECK: !sycl_item_2_1_ = !sycl.item<[2, true], (!sycl.item_base<[2, true], (!sycl.range<2>, !sycl.id<2>, !sycl.id<2>)>)>
```

Comment:
Can you explain why this is not longer being generated ?

Reply:
As per the last commit, ([see](https://github.com/intel/llvm/blob/sycl-mlir/polygeist/tools/cgeist/driver.cc#L989)), if the input contains a kernel, only the device module is generated.